src/dbnode/storage/bootstrap/bootstrapper/fs/source.go

```diff
@@ -436,33 +466,9 @@ func (s *fileSystemSource) markRunResultErrorsAndUnfulfilled(
 		for i := range timesWithErrors {
 			timesWithErrorsString[i] = timesWithErrors[i].String()
```

Comment:
meganit: refactor to use `append`?

Reply:
Might skip refactoring the old code, in preference of expediting merging.